src/models/UsuarioSchema.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
```

Comment:
adicionar linha para separar os grupos de import
```suggestion
import { z } from 'zod';

```